lib/swagger/swaggerTypes.ts

```diff
@@ -260,6 +260,9 @@ export interface Schema extends BaseSchema {
   [refSelfSymbol]?: string;
 
   _skipError?: boolean;
+
+  // x-ms-discriminator-value exists but discriminator missing
+  _missingDiscriminator?: boolean;
```

Comment:
Why record status in swagger type?

Reply:
Because if a schema defined x-ms-discriminator-value but its parent doesn't define discriminator, it won't be record in discriminatorMap, so we can't find it in discriminatorMap. And then in semantic validation, the 'allOf' field is deleted, so we also don't know which is inherited from. So we record the information in the transform.